templates/squid.conf.port.erb

```diff
@@ -1,3 +1,10 @@
-# fragment for <%= @protocol %>_port <%= @port %>
-<%= @protocol %>_port <%= @port %> <%= @options %>
+<%
+  if @_host == nil then
```

Comment:
are you interested in converting this into an epp template? It isn't needed, but highly appreciated.